templates/buildkite-agent.cfg.v3.j2

```diff
@@ -0,0 +1,59 @@
+# The token from your Buildkite "Agents" page
+token="{{ buildkite_agent_token }}"
+
+# The name of the agent
+name="%hostname-%n"
+
+# The priority of the agent (higher priorities are assigned work first)
+priority={{ buildkite_agent_priority }}
+
+# Tags for the agent (default is "queue=default")
+{% set queue = 'queue=' + agent_queue %}
```

Comment:
This is likely meant to reference `buildkite_agent_queue`  vs. `agent_queue`.

Reply:
Yes! Good spot. I think my playbook variable cascaded through and that's why I missed this.